dm/dm.go

```diff
@@ -175,7 +175,11 @@ func main() {
 
 func callService(path, method, action string, reader io.ReadCloser) {
 	u := fmt.Sprintf("%s/%s", *service, path)
-	request, err := http.NewRequest(method, u, reader)
+	callAndPrintHttp(u, method, action, reader)
```

Comment:
I think I'd rather see this separated into calling the service and it returning the body as a string for easier mocking / testing. I don't feel super strong about it though :)